HouseRules_Core/BoardSyncer.cs

```diff
@@ -97,6 +95,7 @@ private static bool CanAbilityEventRepresentNewSpawn(SerializableEventOnAbilityU
                 case AbilityKey.RaiseRoots:
                 case AbilityKey.CallCompanion:
                 case AbilityKey.DigRatsNest:
+                case AbilityKey.MiniBarricade:
```

Comment:
If we're doing MiniBarricade, should we also be doing MagicWall ?

Reply:
Great catch! Fixed.